### DISCUSSION TITLE
Unable to get CNAME or A record for an added domain trough the API

### DISCUSSION THREAD

**Discussion by soufiane-bouchaara** [vercel/community#107] [Help] [answered]
using the API endpoint (https://vercel.com/docs/rest-api#endpoints/domains/get-information-for-a-single-domain),  I am unable to get CNAME or A record for an added domain through the API.
Only name servers is available in the response : 
{
......, 'intendedNameservers': ['ns1.vercel-dns.com', 'ns2.vercel-dns.com']
.....
}

**Comment by soufiane-bouchaara** [comment]
Please , Are there any solutions for this issue ?

**Comment by Pieparker** [comment, marked as answer]
Best thing to do would be to run `dig` in a terminal to find this information. For example:

`dig a vercel.com`
`dig cname www.vercel.com`
